examples/benchmark-ui/requirements.txt

```diff
@@ -0,0 +1,4 @@
+deepsparse-nightly
+deepsparse[onnxruntime]
```

Comment:
is the nightly needed or is 1.4 good enough for now?

Reply:
still need nightly as for right now

Reply:
can you remove `deepsparse[onnxruntime]` and edit `deepsparse-nightly[onnxruntime]`